pkg/agent/datamodel/sig_config_test.go

```diff
@@ -125,7 +125,7 @@ var _ = Describe("GetSIGAzureCloudSpecConfig", func() {
 		Expect(aksUbuntu2204TLGen2Containerd.ResourceGroup).To(Equal("resourcegroup"))
 		Expect(aksUbuntu2204TLGen2Containerd.Gallery).To(Equal("aksubuntu"))
 		Expect(aksUbuntu2204TLGen2Containerd.Definition).To(Equal("2204gen2TLcontainerd"))
-		Expect(aksUbuntu2204TLGen2Containerd.Version).To(Equal("2022.10.13"))
+		Expect(aksUbuntu2204TLGen2Containerd.Version).To(Equal("2023.02.15"))
```

Comment:
Is this the correct image version for 2204gen2TLcontainerd?

Reply:
yes, the tests passed :)